### ISSUE TITLE
Dependencies Error

### ISSUE THREAD

**Issue by emi-dm** [facebookresearch/audioseal#7]
Hi guys! Can you update the requirements file with the correct dependencies? I tried to use the model, but it didn't load the weights correctly. Can you test it? 

Thanks in advance!

**Comment by antoine-tran** [comment]
Hi @emi-dm , could you paste the  errors you get here ?

**Comment by Flux9665** [comment]
I believe audiocraft is missing from the dependencies, which is imported in audioseal.builder

**Comment by Flux9665** [comment]
On Linux it seems to work just fine, on Windows however, I have these issues. I managed to resolve all of them with not too much trouble by hardcoding the path for the config in `load_local_model_config` to a local copy I made of the file and using the URL from the config as input to `load_generator`. Then I also had to set the environment variable `USER` to some dummy value, in order to satisfy OmegaConf.

So since this seems to be a WIndows only issue, I'm not sure if you intend to take care of it, but it would probably not be too difficult.
